### PR TITLE
Sg 22646 shotgrid

### PR DESCRIPTION
**Pull request by zamu5**

### CODE REVIEW

File: python/tk_desktop2/shotgun_entity_path.py [hunk 1]
```diff
@@ -117,9 +117,9 @@ def __init__(self):
     def __repr__(self):
         """String representation"""
         if self.is_valid():
-            return "<ShotgunEntityPath '%s'>" % self.as_string()
```
Comment (jfboismenu) [inline python/tk_desktop2/shotgun_entity_path.py:120]: Please revert.

**Review by jfboismenu — APPROVED** [review]
Looks good to me. @geffrak , anything you can think of that should have been updated as well code wise?

**Comment by jfboismenu** [comment]
One @geffrak gives his ok, you can test @mathurf

**Review by geffrak — APPROVED** [review]
Looks good to me.

**Review by mathurf — APPROVED** [review]
LGTM
Already tested it while helping Sergio. My only concern was about shotgunlocalhost and it has been reverted. So everything is good.

**Pull request merged**